main/src/com/google/refine/importers/tree/XmlImportUtilities.java

```diff
@@ -265,13 +268,13 @@ static public void importTreeData(
             }
         } catch (TreeReaderException e) {
             logger.error("Exception from XML parse",e);
-            throw e;
+            throw new ImportingParserException(((JsonParseException)e.getCause()).getOriginalMessage());
```

Comment:
I don't think we can assume that all `TreeReaderException` instances are `JsonParseException`… Especially since `JsonParseException` is not a subclass of `TreeReaderException`! It would be good to have a unit test to show how this works. (This sort of mistake should be caught by any unit test covering these lines)